tests/Extensions/Hosting.Services.UnitTests/HostBuilderExtensionsTests.cs

```diff
@@ -68,30 +67,30 @@ public void AddServiceActionUsingObject_RegisterInstance()
 		public async Task AddServiceActionUsingFunc_RegisterInstance()
 		{
 			bool actionCalled = false;
-			Func<ServiceContext, CancellationToken, Task> action = (c, t) =>
+			Task action(object c, CancellationToken t)
 			{
 				actionCalled = true;
 				return Task.CompletedTask;
-			};
+			}
 
 			HostBuilder hostBuilder = new HostBuilder();
-			new ServiceFabricHostBuilder<ServiceContext>(hostBuilder).AddServiceAction(action);
+			new ServiceFabricHostBuilder<object>(hostBuilder).AddServiceAction(action);
 
-			IServiceAction<ServiceContext> resolvedAction = hostBuilder.Build().Services.GetService<IServiceAction<ServiceContext>>();
+			IServiceAction<object> resolvedAction = hostBuilder.Build().Services.GetService<IServiceAction<object>>();
 
-			await resolvedAction.RunAsync(MockStatelessServiceContextFactory.Default, CancellationToken.None);
+			await resolvedAction.RunAsync(new object(), CancellationToken.None);
```

Comment:
>RunAsync [](start = 24, length = 8)

Consider adding `.ConfigureAwait(false)`. Applies thrroughout

Reply:
Added it, thank you